packages/cms-lib/lib/watch.js

```diff
@@ -56,9 +98,43 @@ function uploadFile(portalId, file, dest, { mode, cwd }) {
   });
 }
 
-function watch(portalId, src, dest, { mode, cwd, remove, disableInitial }) {
+async function deleteRemotePath(portalId, filePath, remotePath, { cwd }) {
```

Comment:
I think that the name of this function should have "file" in its name since it only applies to files and not directories.